app/helpers/_flashes_helper.rb

```diff
@@ -0,0 +1,22 @@
+module FlashesHelper
+
+  ACCEPTED_FLASH_HTML_CLASSES = {
+    success: 'alert alert-success',
+    error: 'alert alert-danger'
```

Comment:
Is there any interest in adding the two other Bootstrap alert styles at this point?
Do we want alerts to be dismissible?